app/scripts/model/dataProxy.js

```diff
@@ -298,8 +308,14 @@ window.DataManagerForIviz = (function($, _) {
                     sequencedCases: _caseLists.allSampleIds
                   };
                 }
-                if (['CANCER_TYPE', 'CANCER_TYPE_DETAILED'].indexOf(_metaObj.attr_id) !== -1) {
-                  _metaObj.priority = 0.9;
+                if (['CANCER_TYPE', 'CANCER_TYPE_DETAILED']
+                    .indexOf(_metaObj.attr_id) !== -1) {
+                  if (Object.keys(_studyToSampleToPatientMap)
+                      .indexOf('MSKIMPACT') === -1) {
+                    _metaObj.priority = 4.1;
```

Comment:
These priorities are hard coded( Changing priority depending on the study). May be we need to come up with a better solution in future release.